### DISCUSSION TITLE
Bebês Velhinhos

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#432] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/bebes-velhinhos/

**Comment by brunoaugusto** [comment]
Alguns até parecem com personalidades famosas, tipo eu vejo o primeiro e a primeira pessoa que me vem à mente é o Boris Casoy

**Comment by one-eyed-willy** [comment]
> [Bruno Augusto](</perfil/?user=brunoaugusto>) disse [segunda-feira, 20 de junho de 2022](<https://www.vish.blog.br/imagens/bebes-velhinhos/#comment_1160590412>):

> Alguns até parecem com personalidades famosas, tipo eu vejo o primeiro e a primeira pessoa que me vem à mente é o Boris Casoy

Eu achei o quinto a cara do Castrinho ou talvez do Charles Paraventi

**Comment by angellisy** [comment]
> [Willy Caolho](</perfil/?user=one-eyed-willy>) disse [segunda-feira, 20 de junho de 2022](<https://www.vish.blog.br/imagens/bebes-velhinhos/#comment_1160642742>):

> Eu achei o quinto a cara do Castrinho ou talvez do Charles Paraventi

Achei o décimo-sétimo bem parecido com o ator Brian Baumgartner:



![null](<https://i.imgur.com/eKnzbdG.png>)

**Comment by maryon-telvanni** [comment]
> [angellisy](</perfil/?user=angellisy>) disse [segunda-feira, 20 de junho de 2022](<https://www.vish.blog.br/imagens/bebes-velhinhos/#comment_1160672823>):

> Achei o décimo-sétimo bem parecido com o ator Brian Baumgartner:[![null](<https://camo.githubusercontent.com/00a1fbc8762c43f354b162c82bb3c93f3fc06857c0d540e739a91bf46e162ca0/68747470733a2f2f692e696d6775722e636f6d2f654b6e7a6264472e706e67>)](<https://camo.githubusercontent.com/00a1fbc8762c43f354b162c82bb3c93f3fc06857c0d540e739a91bf46e162ca0/68747470733a2f2f692e696d6775722e636f6d2f654b6e7a6264472e706e67>)

Com todo respeito ao ator, mas, "The Office"? Cruzes!
